docs/manifests.md

```diff
@@ -0,0 +1,30 @@
+# Jotting notes here
```

Comment:
```suggestion
# TODO Jotting notes here
```

I'm throwing in "TODO" wherever we have draft text that we'll definitely want to replace in a final polished version so we can easily search for it